subgraphs/uniswap-forks/protocols/pangolin/config/deployments/pangolin-avalanche/configurations.json

```diff
@@ -0,0 +1,13 @@
+{
+  "deployment": "PANGOLIN",
```

Comment:
Should be `PANGOLIN_AVALANCHE`. Also, you should add the Pangolin avalanche configurations to configurations/configurations.ts

Reply:
Thanks have changed to this